libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/NativeLoginManager.kt

```diff
@@ -270,108 +266,194 @@ internal class NativeLoginManager(
         return requestBodyString.toRequestBody(mediaType)
     }
 
-    // region Headless, Password-Less Login Via One-Time-Passcode
+    // region Salesforce Identity API Headless Forgot Password Flow
 
-    override suspend fun submitOtpRequest(
+    override suspend fun startPasswordReset(
```

Comment:
Now that I'm getting used to these two-step identity flows, I decided on a really clear "start" and "complete" naming prefix.  That seems to make it really obvious which method goes where in the app's control flow.